backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -169,16 +169,17 @@ async def upload_boundaries_excel_sheet(
+        input_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
+        content = await boundary_file.read()
+        input_temp_file.write(content)
+        input_temp_file.close()
+        boundary_file_path = input_temp_file.name
 
         timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
         output_filename = f"boundary_validation_results_{timestamp}.xlsx"
-
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx") as output_temp_file:
-            output_file_path = output_temp_file.name
+        output_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)